docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -125,4 +126,23 @@ When the {agent} is running you can add a new private location in {kib}:
 
 Now you can add monitors to your private location in <<synthetics-get-started-ui,the {uptime-app}>> or using the <<synthetics-get-started-project,Elastic Synthetics library's `push` method>>.
 
-TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
\ No newline at end of file
+TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
+
+[discrete]
+[[synthetics-private-location-scaling]]
+= Scaling private locations
+
+By default private locations are configured to allow two simultaneous browser tests, and an unlimited number of lightweight checks. 
+These limits can be set via the environment variables `SYNTHETICS_LIMIT_{TYPE}`, where `{TYPE}` is one of `BROWSER`, `HTTP`, `TCP`, and `ICMP` 
+for the container running the {agent} docker image. 
+
+It is critical to allocate enough memory and CPU capacity to handle configured limits. 
+Start by allocating at 2 GiB of memory and two cores per browser instance to ensure consistent
```

Comment:
Typo?
>Start by allocating at 2 GiB of memory

Start by allocating at _least_ 2 GiB of memory

Reply:
```suggestion
Start by allocating at least 2 GiB of memory and two cores per browser instance to ensure consistent
```